sip-core/src/test/resources/test-factory.yaml

```diff
@@ -1,3 +1,3 @@
 sip1:
   hierarchy: test1
-sip2.hierarchy: test2
+sip2.hierarchy: test2
```

Comment:
If we don't need it I would remove it!

Reply:
I am not sure why this appeared here, since it is a part of a test for different functionality. And the line is not changed also.